dll/shellext/netplwiz/netplwiz.c

```diff
@@ -0,0 +1,38 @@
+/*
+ * COPYRIGHT:       See COPYING in the top level directory
+ * PROJECT:         ReactOS Disconnect Network Drive
+ * FILE:            dll/shellext/netplwiz/netplwiz.c
+ * PURPOSE:
+ *
+ * PROGRAMMERS:     Jared Smudde
+ */
+
+#include "netplwiz.h"
+
+HRESULT WINAPI DllCanUnloadNow(VOID)
+{
+    return S_OK;
+}
+
+HRESULT WINAPI DllGetClassObject(REFCLSID rclsid, REFIID riid, LPVOID *ppv)
+{
+    *ppv = NULL;
+    return E_NOTIMPL;
+}
+
+BOOL WINAPI DllMain(HINSTANCE hinstDLL, DWORD dwReason, LPVOID lpvReserved)
+{
+    INITCOMMONCONTROLSEX iccx;
+    hInstance = hinstDLL;
+    switch (dwReason)
+    {
+        case DLL_PROCESS_ATTACH:
+            iccx.dwSize = sizeof(INITCOMMONCONTROLSEX);
+            iccx.dwICC = ICC_STANDARD_CLASSES | ICC_LISTVIEW_CLASSES;
+            InitCommonControlsEx(&iccx);
```

Comment:
Calling a comctl32 function from DllMain seems problematic. Is there an alternative place you could move it to?
(and yes, I know we have other dlls/cpls that do this. Doesn't mean it's a good idea)

Reply:
Should I leave it in place or did you all want me to move it?